resources/js/features/home/components/+sidebar/AchievementOfTheWeek/AchievementOfTheWeek.tsx

```diff
@@ -54,15 +63,30 @@ export const AchievementOfTheWeek: FC = () => {
 
               <div className="flex flex-col gap-0.5">
                 <GameAvatar {...game} showImage={false} />
-                <SystemChip {...system} className="bg-zinc-800" />
+                <div className="flex items-center">
+                  <SystemChip {...system} className="bg-zinc-800" />
+
+                  {achievementOfTheWeek.activeUntil ? (
+                    // TODO: make this align bottom right
+                    <div className="w-ful flex-end justify-end">
+                      <span className="smalldate">
+                        <span>{t('Ends')} </span>
+                        <DiffTimestamp
+                          at={achievementOfTheWeek.activeUntil}
+                          asAbsoluteDate={auth?.user.preferences.prefersAbsoluteDates}
+                        />
+                      </span>
+                    </div>
+                  ) : null}
+                </div>
               </div>
```

Comment:
This is pretty cool. I'm looking forward to seeing this go live. Happy to help button this up.

First, let's fix the CSS thing. This was kinda tricky to solve. Here's the solution:
```tsx
<div className="flex w-full flex-col gap-0.5">
  <GameAvatar {...game} showImage={false} />

  <div className="flex w-full items-center justify-between">
    <SystemChip {...system} className="bg-zinc-800" />

    {achievementOfTheWeek.activeUntil ? (
      <span className="smalldate">
        {/* TODO see below */}
      </span>
    ) : null}
  </div>
</div>
```

Next, for the `smalldate` content, this is a circumstance we'll need to reach for `<Trans />` to interpolate the value rather than concatenate it. In SOV languages and some of our top 10 locales by traffic, the timestamp will come before the label, so we'll need to facilitate that working correctly.

First, in _en_US.json_, adjust to:
```json
"Ends <1>{{when}}</1>": "Ends <1>{{when}}</1>"
```

The `<Trans />` syntax is kinda weird. Here's the solution:

```tsx
<span className="smalldate">
  <Trans
    i18nKey="Ends <1>{{when}}</1>"
    values={{ when: achievementOfTheWeek.activeUntil }}
    components={{
      1: (
        <DiffTimestamp
          at={achievementOfTheWeek.activeUntil}
          asAbsoluteDate={auth?.user.preferences.prefersAbsoluteDates}
        />
      ),
    }}
  />
</span>
```

In the UI, you should now see "Ends [some diff]". To prove that `<Trans />` is working correctly, go back to _en_US.json_ and adjust to:
```json
"Ends <1>{{when}}</1>": "<1>{{when}}</1> Ends"
```
Now when you refresh the UI, you'll see "[some diff] Ends".

Working with these interpolations takes some getting used to. I'm always happy to help with these as they come up.